src/VS.Web.CG.Mvc/Common/RequiredFileEntity.cs

```diff
@@ -23,6 +24,15 @@ public RequiredFileEntity(string outputPath, string templateName)
             OutputPath = outputPath;
         }
         
+        public RequiredFileEntity(string outputPath, string templateName, List<string> altPaths)
```

Comment:
Minor: Accepting `IEnumerable<string>` instead would simplify the work of the callers.